modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+// * observability/network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-netobserv-cli-commands_{context}"]
+= Network Observability CLI commands
+
+.Syntax
+`netobserv [flows|packets|cleanup] [filters]`
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+|Option| Description
+
+| `flows`
+| Capture flows information. You can specify an optional interface name as a filter, such as 'netobserv flows br-ex'.
+
+| `packets`
+| Capture packets from a specific protocol or port pair, such as 'netobserv packets tcp,80'.
+
+| `cleanup`
+| Remove netobserv components.
+
+| `version`
+| Print the software version.
+
+| `--help`
+| Show help.
+|===
+
+
+
+.Network Observability feature enablement
```

Comment:
all the following options are `flows` subcommands as showing in the help
no relevant for other main options

Reply:
Gotcha, yeah I think I need to put a header here and maybe mention in the Basic commands table that for more information about Flows sub-commands, see this table.